src/config/dynamic-theme-fixes.config

```
@@ -14379,6 +14379,15 @@ INVERT
 
 ================================
 
+overreacted.io
+
+CSS
+div[style*="background: var(--bg)"] {
```

```suggestion
:root {
```

No need to specify this variable that deeply.

---

Understood.